test/OpenTelemetry.Instrumentation.Http.Tests/HttpWebRequestActivitySourceTests.netfx.cs

```diff
@@ -413,8 +413,10 @@ public async Task TestTraceStateAndBaggage()
         [Theory]
         [InlineData("GET")]
         [InlineData("POST")]
-        public async Task DoNotInjectTraceParentWhenPresent(string method)
+        public async Task CreateSpansIfTraceParentIsPresent(string method)
         {
+            const string traceId = "abcdef0123456789abcdef0123456789";
+            const string parentSpanId = "abcdef0123456789";
```

Comment:
same comment as .net core test.. please name it either parentId or SpanId.

Reply:
Done in [dfe3520](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3072/commits/dfe352066a7383b4c679ee8bcb72cd34d472b93b)